hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/PartitionManagerImpl.java

```diff
@@ -183,6 +186,19 @@ public boolean isDistributed() {
         return distributed;
     }
 
+    @Override
+    public Map<String, String> getStatus() {
+        if (distributed) {
+            /*
+                Members are dynamic, so on each status request this info is refreshed
+             */
+            status.put("members", cacheManager.getMembers().stream()
```

Comment:
Block comments are so C :)   Aside from a preference for //, this is pretty cool, it looks like you are able to actually get member info, and not just a member count, that will be very useful, I think.